sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -1506,6 +1534,39 @@ def process_bundle(self, inputs, expected_outputs):
     return result, split_results
 
 
+class ParallelBundleManager(BundleManager):
+
+  def process_bundle(self, inputs, expected_outputs, num_workers=None):
```

Comment:
Do we need two separate ways of passing num_workers?

Reply:
I added it in case we want to change num_workers for a certain bundle only, instead of changing at BundleManager level. Do we have a case that process multi bundles parallel with different num_workers? I now changed back to support only self._num_workers, I would like to grab your advice if we even need to consider such cases.

Reply:
While I wouldn't rule out finding a reason to support such a thing later, I would say that it's be cleaner to add such support if/once we need it rather than now. It's generally much easier to read/follow code if parameters come from only one place.